### PR TITLE
FACT-579 Updated opening hours tab with revised form for adding new entries

### PR DESCRIPTION
**Pull request by agraham01**
https://tools.hmcts.net/jira/browse/FACT-579

Updated opening hours tab with revised form for adding new entries

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/views/courts/tabs/openingHoursContent.njk [hunk 1]
```diff
@@ -37,19 +37,27 @@
 
 {% for openingTime in opening_times %}
 
+  {% set isNew = (openingTime.isNew === true) or (openingTime.isNew === "true") %}
+
   {% set descriptionError = false %}
   {% if openingTime.type_id === '' %}
-    {% set descriptionError = "{ text: 'Type is required' }" %}
+    {% set descriptionError = { text: 'Type is required' } %}
```
Comment (Thomas-Geraghty) [inline src/main/views/courts/tabs/openingHoursContent.njk:44]: Is the wrapping with "" not necessary? Just wondering as I saw @Majid-justice add these onto one of his amendments for the tabs he has worked on. Unsure which is correct without diving in and testing myself.

File: src/test/functional/features/opening-hours.feature [hunk 2]
```diff
@@ -7,30 +7,30 @@ Feature: Opening Hours
...
+    When I enter an incomplete opening hours entry
     And I click save
     Then an error message is displayed
 
   Scenario: Remove opening hours
     When I click the remove button under an opening hours entry
     Then I click save
-    Then a green update message is displayed
+    Then a green update message is displayed in the opening hours tab
+    # Delete another to remove both entries added above
```
Comment (Thomas-Geraghty) [inline src/test/functional/features/opening-hours.feature:33]: Does this comment need to stay?

File: src/main/views/courts/tabs/openingHoursContent.njk (hunk 1 above)
Comment (agraham01) [reply src/main/views/courts/tabs/openingHoursContent.njk:44]: This change is correct. Currently, the error message is not displaying next to the input element and this fixes it.

File: src/test/functional/features/opening-hours.feature (hunk 2 above)
Comment (agraham01) [reply src/test/functional/features/opening-hours.feature]: I've removed this comment.

File: src/main/views/courts/tabs/openingHoursContent.njk (hunk 1 above)
Comment (Thomas-Geraghty) [reply src/main/views/courts/tabs/openingHoursContent.njk:44]: Awesome, cheers for clarifying @agraham01

**Review by Thomas-Geraghty — APPROVED** [review]

**Pull request merged**